internal/tmpl/character/helper.go

```diff
@@ -1,9 +1,20 @@
 package character
 
+import "github.com/genshinsim/gcsim/pkg/core"
+
 //advance normal index, return the current one
 func (c *Tmpl) AdvanceNormalIndex() {
 	c.NormalCounter++
 	if c.NormalCounter == c.NormalHitNum {
 		c.NormalCounter = 0
 	}
 }
+
+// c.NormalCounter returns the next hit, and always resets
+// when it reaches the max hit of the normal attack combo
+func (c *Tmpl) NextNormalCounter() int {
```

Comment:
sorry i'm a bit confused on this one. Isn't c.NormalCounter already the next normal index?